README.md

```diff
@@ -6,66 +6,77 @@
-   data cache implementation using [Stash](http://www.stashphp.com/Drivers.html) and binary file system handled by [Flysystem](https://github.com/thephpleague/flysystem#adapters).
-   Improved *Storage Engine* planned, custom implementation for increased data scalability already possible.
- - Powerful (& extensible) Content Query engine, currently powered by SQL, soon Solr/ElasticSearch
- - Very high performance thanks to content & user aware HTTP <sup>"view"</sup> cache (now [using](https://github.com/FriendsOfSymfony/FOSHttpCacheBundle))
- - Introduces concept of "web sites" allowing you to manage several within one installation
- - Allows to rapidly set up *Contextual override* of content display twig templates as well as controller based on Content type, section, and much more.
- - Helpers, services, events and signals allowing you to efficiently create everything from simple web sites to complex applications
-- **PlatformUIBundle**: A modern, extensible "Backend" UI for managing content & administering the site
-
-### Further information:
-eZ Platform is 100% open source and is released as companion to a commercial *eZ Studio* software which adds advanced
-features for editorial teams and media companies, 100% built on top of *eZ Platform* APIs.
-
-- [eZ Publish 5.x Architecture, including "Platform Stack"](https://doc.ez.no/pages/viewpage.action?pageId=11403666)
-- [eZ Studio and eZ Platform RoadMap](http://ez.no/Blog/What-to-Expect-from-eZ-Studio-and-eZ-Platform)
-- [eZ Platform 2015 release plan](http://ez.no/Blog/What-Releases-to-Expect-from-eZ-in-2015)
-- eZ Systems AS *(commercial products and services)*: [ez.no](http://ez.no/)
-- eZ Community: [share.ez.no](http://ez.no/)
-
-## Install and Upgrade
-For installation & upgrade instructions, see [INSTALL.md](https://github.com/ezsystems/ezplatform/blob/master/INSTALL.md).
+## What is eZ Platform Demo ?
+
+*eZ Platform Demo* is an example of a simple website using *eZ Platform*. This example should not be use for starting a new project.
+
+Features:
+- Blog
+- Gallery
+- Contact Form
 
 ## Requirements
-**eZ Platform** currently has the same requirements as *eZ Publish Platform 5.4*, further details on the [5.4 requirements](https://doc.ez.no/display/EZP/Requirements+5.4) page.
+**eZ Platform** currently has the same requirements as *eZ Publish Platform 5.4*, further details on the [requirements](https://doc.ez.no/display/TECHDOC/Requirements) page.
 
 *TL;DR: minimum PHP 5.4.4 and higher, using mod_php or php-fpm.*
 
+## Installation
+
+### 1. Create a database
+
+First, create a new database using the following command:
+
+```bash
+mysql> CREATE DATABASE <database> CHARACTER SET utf8;
+```
+
+### 2. Clone ezplatform-demo
+
+Clone the Github repository containing the demo:
+
+```bash
+git clone https://github.com/ezsystems/ezplatform-demo
+cd ezplatform-demo
+```
+
+### 3. Install dependencies
+
+Run `composer install` to install the required dependencies:
```

Comment:
Should be composer install I think, or 2 and 3 can be combined by using composer create-project once this repo exists on packagist.